qt/applications/workbench/workbench/plotting/plotscriptgenerator/axes.py

```diff
@@ -53,26 +53,23 @@ def generate_axis_scale_commands(ax):
     return commands
 
 
-def generate_tick_commands(ax):
-    commands = []
-    if not isinstance(ax.xaxis.minor.locator, NullLocator):
-        commands.append("axes.minorticks_on()")
+def generate_tick_params_kwargs(axis, tick_type="major"):
+    return getattr(axis, f"_{tick_type}_tick_kw")
 
-        if hasattr(ax, 'show_minor_gridlines'):
-            commands.append(f"axes.show_minor_gridlines = {ax.show_minor_gridlines}")
 
-    return commands
+def generate_tick_commands(ax):
+    commands = []
 
+    for tick_type in ["minor", "major"]:
+        if not isinstance(getattr(ax.xaxis, tick_type).locator, NullLocator):
+            if tick_type == "minor":
+                commands.append("axes.minorticks_on()")
```

Comment:
If you try and do a tile plot with minor ticks on, this line cannot be run, presumably as it should be axes[0][x] for each plot

Reply:
Is this a feature regression from release-next/last-release? or is this a new issue because line 64 is largely the same as the way it was previously.

Reply:
If it is a regression I will fix it, if not I will note it and fix it in an upcoming Epic.

Reply:
Looks like it is in 5.1 as well and should be added to the epic